readme.md

```diff
@@ -480,6 +480,7 @@
 - [Software-Defined Networking](https://github.com/sdnds-tw/awesome-sdn)
 - [Network Analysis](https://github.com/briatte/awesome-network-analysis)
 - [PCAPTools](https://github.com/caesar0301/awesome-pcaptools)
+- [Computer Networking](https://github.com/nyquist/awesome-networking)
```

Comment:
Maybe add a description to the list?

Reply:
Hi,
I followed the format of the other items in the list. The description is in the list itself, but if needed I can add it.

Reply:
Yes, should have a description.